src/pages/overview/pdf-embed-api/index.md

```diff
@@ -151,16 +151,24 @@ and select the desired view mode from the dropdown.
 
 **Continuous view mode**
 
-![image](../images/continuous_mode.png)
```

Comment:
Please remove the usage of image text from here. 
It was a placeholder used to specify alt text of the image. 

Please take a look here and modify the changes https://github.com/AdobeDocs/pdfservices-api-documentation/pull/164/files